python/example_code/rds/README.md

```diff
@@ -76,6 +81,13 @@ Sample applications that work across multiple AWS services.
 <!--custom.instructions.start-->
 <!--custom.instructions.end-->
 
+#### Hello Amazon RDS
+
+This example shows you how to get started using Amazon RDS.
+
+```
+python hello.py
```

Comment:
Is this command correct, shouldn't it use the new filename? Same question throughout, new filenames means the command should be updated.

Reply:
This is actually a widespread problem with all of the hello examples generated by the README, due to a conflict in the spec and the way WRITEME works. Looking into it now.

Reply:
Fixed